### PR TITLE
feat: DEV-2755: Regions list grouped by labels with persistent & ordered collapse

### PR DESCRIPTION
**Pull request by yyassi-heartex**

### CODE REVIEW

**Review by juliosgarbi — CHANGES_REQUESTED** [review]
Features must be behind feature flags also there is some inconsistency in the spacing between '( )' some places have some not

**Review by juliosgarbi — DISMISSED** [review]
LGTM!

**Review by juliosgarbi — APPROVED** [review]
LGTM

File: src/components/Entities/LabelList.js [hunk 1]
```diff
@@ -19,6 +43,8 @@ export const LabelList = observer(({ regionStore }) => {
     };
   });
 
+  const expandedKeys = treeData.filter( item => !collapsedPos.includes( item.pos ) ).map( item => item.key ) ?? [];
```
Comment (bmartel) [inline src/components/Entities/LabelList.js:46]: probably should put any of the new additional functionality behind the FF.

the `Tree` props `expandedKeys` and `onExpand` will have the new behaviour by default right now.

**Review by bmartel — CHANGES_REQUESTED** [review]

**Review by bmartel — APPROVED** [review]

**Pull request merged**